apps/rush-lib/src/api/ApprovedPackagesConfiguration.ts

```diff
@@ -47,9 +47,9 @@ export class ApprovedPackagesItem {
  * @public
  */
 export class ApprovedPackagesConfiguration {
-  private static _jsonSchema: JsonSchema = JsonSchema.fromFile(
-    path.join(__dirname, '../schemas/approved-packages.schema.json')
-  );
+  private static get _jsonSchema(): JsonSchema {
+    return JsonSchema.fromFile(path.join(__dirname, '../schemas/approved-packages.schema.json'));
```

Comment:
This will hurt runtime performance, because now we will call `JsonSchema.fromFile()` every single time the property is accessed.  We need to cache the value in a local variable.  The `SomeClass.jsonSchema` property is a common pattern, so we should standardize the pattern and use it everywhere.

(BTW originally we wanted all the schemas to get validated on load by design, to proactively catch any parse errors.  But that design definitely seems too expensive now that we're optimizing perf.)

Reply:
Hey, maybe the `JsonSchema` API can do the delay loading optimization automatically? That would be  a cool solution.